doc/source/conf.py

```diff
@@ -47,7 +53,7 @@
               'sphinx.ext.mathjax',
               'sphinx.ext.inheritance_diagram',
               'sphinx.ext.autosummary',
-              'math_dollar', # has to go before numpydoc
+              'math_dollar',  # has to go before numpydoc
               # we have a local copy of the extension, imported from NumPy 1.3
               # this also includes the docscrape* extensions
               'numpydoc',
```

Comment:
I was thinking we could use the relevant pip packages here:

```
pip install numpydoct texext
```

Then add `texext.math_dollar, numpydoc` to `extensions`.  We could then delete both those extensions from the `sphinxext` directory.